functions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+function dbConnect(): PDO
+{
+    $db = new PDO('mysql:host=db;dbname=collection_app', 'root', 'password');
+    $db->setAttribute(PDO:: ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    return $db;
+}
+
+function getData(PDO $db):array
+{
+    $query = $db->prepare('SELECT `name`, `painting_title`, `type`, `description`, `type` , `image` 
+       FROM `painting` JOIN `artist` ON `painting`.`artist_name` = `artist`.`id`;');
+    $query->execute();
+    $result = $query->fetchAll();
+
+    return $result;
+}
+
+function extractDataAndDisplay(array $db):string
+{
+    $text = '<section class="container-box">';
+    foreach ($db as $key) {
```

Comment:
Please improve naming, maybe change `$key` to `$row`, and change `$db` to `$data` or to `$artCollection`. Optional, I will pass this code review without it.